.travis.yml

```diff
@@ -5,6 +5,7 @@ sudo: false
 cache: packages
 r_packages:
   - covr
+  - gert
```

Comment:
Just curious - why is `gert` required by travis?

Reply:
I misunderstood .travis.yml when I was trying to get libgit2 installed! Removed...